python/hsfs/feature_view.py

```diff
@@ -804,6 +804,7 @@ def get_batch_data(
         primary_keys=False,
         event_time=False,
         inference_helper_columns=False,
+        dataframe_type: Optional[str] = "default",
```

Comment:
This is user interfacing function, is it possible to avoid  `dataframe_type = "default"` or explain clearly what `"default"` means